yesod-test/Yesod/Test.hs

```diff
@@ -10,7 +10,7 @@
 
 {-|
 Yesod.Test is a pragmatic framework for testing web applications built
-using wai and persistent.
+using wai.
```

Comment:
yesod-test doesn't really have any direct connection to Persistent, and I didn't want to give people the impression that you couldn't use it with another database library.

Possibly worth mentioning the scaffolding uses Persistent to show that it is compatible with Persistent though?

Reply:
Nah, I think this is fine as-is.